src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Route, Switch ,withRouter } from 'react-router-dom';
+import Layout from './hoc/Layout/Layout';
+import postRoute from './hoc/asyncComponent';
+import { connect } from 'react-redux';
+import * as actions from './store/actions/index';
+
+const addPost = postRoute(() => {
+  return import('./containers/Posts/Posts')
+})
+
+const auth = postRoute(() => {
+  return import('./containers/Auth/Auth')
+})
+
+const logout = postRoute(() => {
+  return import('./containers/Auth/Logout/Logout')
+})
+
+const fullPost = postRoute(() => {
+  return import('./containers/Posts/FullPost/FullPost')
+})
+
+const postList = postRoute(() => {
+  return import('./containers/Posts/PostList/PostList')
```

Comment:
Can you take route meta as an array and import that dynamically in Route section

Reply:
working on this.